docs/core/diagnostics/snippets/exception-summary/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Net;
+using System.Net.Sockets;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.ExceptionSummarization;
+
+// Add exception summarization services.
+var services = new ServiceCollection()
+    .AddExceptionSummarizer(static builder => builder.AddHttpProvider());
+
+var provider = services.BuildServiceProvider();
+
+// Get the exception summarizer.
+var summarizer = provider.GetRequiredService<IExceptionSummarizer>();
+
+foreach (var exception in new Exception[]
+    {
+        new OperationCanceledException("Operation cancelled..."),
+        new TaskCanceledException("Task cancelled..."),
+        new SocketException(10_024, "Too many sockets open..."),
+        new WebException("Keep alive failure...",
+            WebExceptionStatus.KeepAliveFailure)
+    })
```

Comment:
I would keep this as static array of exceptions instead of allocating inline. I think that inline allocation in docs is teaching new developers wrong patterns (this code will be probably copied a lot).